backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/edges/AbstractEdgeDescriptionProvider.java

```diff
@@ -62,6 +63,15 @@ protected boolean isDeletable() {
         return true;
     }
 
+    /**
+     * Implementers can override this method to provide a specific label edit tool to apply on the edge.<br>
+     * By default no edit tool is provided.
+     * @return a label edit tool or {@code null} if no edit tool needed.
```

Comment:
sometimes you use `{@code null}` and sometimes `<code>null</code>`. Is there a specific reason?
If not, could you juste use `<code>null</code>` everywhere please?

Reply:
Done